etc/kayobe/kolla.yml

```diff
@@ -309,6 +309,19 @@ kolla_build_blocks:
                -e '/\[{{ repo.tag }}\]/,/^\[/ s|^\(name.*\)|\1\nbaseurl={{ repo.url }}|' /etc/yum.repos.d/{{ repo.file }}{% if not loop.last %} &&{% endif %} \
     {% endfor %}
     {% endif %}
+  base_debian_after_sources_list: |
+    RUN echo "\
+    deb http://ubuntu-cloud.archive.canonical.com/ubuntu jammy-updates/yoga main"\
```

Comment:
Looks like jammy only supports zed+: http://ubuntu-cloud.archive.canonical.com/ubuntu/dists/jammy-updates/

Reply:
I think the version I want (2.17.7) is in  the focal ones after all:

Reply:
Have changed back to uca focal-updates

Reply:
It doesn't seem to be present as a Focal package: https://launchpad.net/ubuntu/focal/+source/openvswitch

Reply:
The uca is using the jammy package 2.17.7-0ubuntu0.22.04.1~cloud1
https://launchpad.net/ubuntu/jammy/+source/openvswitch

Reply:
The Kolla Ubuntu images should have the yoga UCA repos enabled. This config was used previously to pull in the yoga UCA repos on wallaby.

Reply:
Oh ok, that's good news. I wasn't aware that the UCA was referenced by Kolla already. In that case, I'll set of a kolla container build for openvswitch and close this PR.